doc/source/reference/routines.other.rst

```diff
@@ -55,4 +55,11 @@ Matlab-like Functions
    :toctree: generated/
 
    who
-   disp
\ No newline at end of file
+   disp
+
+Exceptions
+----------
+.. autosummary::
+   :toctree: generated/
+
+   AxisError
```

Comment:
A few other candidates the might be worthwhile to add here:
https://github.com/numpy/numpy/blob/89babbd98fc75399709543f55013ff856280c668/numpy/__init__.pyi#L3646-L3653